frontend/lib/src/components/shared/Dropdown/VirtualDropdown.test.tsx

```diff
@@ -29,28 +31,29 @@ function Option(props: OptionProps): ReactElement {
 
 describe("VirtualDropdown element", () => {
   it("renders a StyledEmptyState when it has no children", () => {
-    const wrapper = shallow(<VirtualDropdown />)
+    render(<VirtualDropdown />)
 
-    expect(wrapper.find("StyledEmptyState").exists()).toBeTruthy()
+    expect(screen.getByTestId("stVirtualDropdownEmpty")).toBeInTheDocument()
   })
 
   it("renders a StyledEmptyState when it has children with no item", () => {
-    const wrapper = shallow(
+    render(
       <VirtualDropdown>
         <Option />
       </VirtualDropdown>
     )
 
-    expect(wrapper.find("StyledEmptyState").exists()).toBeTruthy()
+    expect(screen.getByTestId("stVirtualDropdownEmpty")).toBeInTheDocument()
   })
 
   it("renders a FixedSizeList when it has children", () => {
-    const wrapper = mount(
+    render(
       <VirtualDropdown>
         <Option item={{ value: "abc" }} />
       </VirtualDropdown>
     )
 
-    expect(wrapper.find("FixedSizeListItem").exists()).toBeTruthy()
+    // each option will have a tooltip attached to it
```

Comment:
Can you also add check below for completeness:
```
expect(screen.getByTestId("stVirtualDropdown")).toBeInTheDocument()
```

Reply:
yep